brainstormforce/astra
# AST-3379 - "Post Comment" button compatibility with customizer button styling & Guillemet symbol ">>" from "Post Comment" removed.

### Description

- Fix: "Post Comment" button compatibility with customizer button styling.
- Improvement: Guillemet symbol ">>" from "Post Comment" removed.

### Screenshots

https://share.getcloudapp.com/wbuLBOQm

### Types of changes




Bug fixes and minor improvement

### How has this been tested?

- Check if all the below styles apply correctly in frontend, and customizer preview as expected:
- Button presets
- Color Settings
- Font Styles
- Padding
- Border Width
- Also check relevant values for responsive cases.
- Check if the Gulliments “>>” icon is removed as expected.
- Same should be tested for Search Button.

### Checklist:
